src/frontend/Tactics.mli

```diff
@@ -10,9 +10,12 @@ module R := Refiner
 
 open Tactic
 
+val is_total : D.sign -> [`TotalAll | `TotalSome | `NotTotal] RM.m
```

Comment:
A short comment here describing the meaning of each of the return types would be handy :)

Reply:
done